ledger-core/virtual/integration/method_test.go

```diff
@@ -70,6 +70,47 @@ func Method_PrepareObject(ctx context.Context, server *utils.Server, state paylo
 	server.WaitActiveThenIdleConveyor()
 }
 
+func tolerableFlags() contract.MethodIsolation {
+	return contract.MethodIsolation{
+		Interference: contract.CallTolerable,
+		State:        contract.CallDirty,
+	}
+}
+
+func intolerableFlags() contract.MethodIsolation {
+	return contract.MethodIsolation{
+		Interference: contract.CallIntolerable,
+		State:        contract.CallDirty,
+	}
+}
+
+// for happy path tests
+func assertCalleeResponse(t *testing.T, res *payload.VCallResult, objectCaller reference.Global, flagsCaller contract.MethodIsolation) {
+	require.Equal(t, payload.CTMethod, res.CallType)
+	require.Equal(t, objectCaller, res.Caller)
+	assert.Equal(t, flagsCaller.Interference, res.CallFlags.GetInterference()) // copy from VCallRequest
+	assert.Equal(t, flagsCaller.State, res.CallFlags.GetState())
+}
+
+// for happy path tests
+func assertCallerResponse(t *testing.T, res *payload.VCallResult, server *utils.Server, flagsCaller contract.MethodIsolation) {
+	require.Equal(t, payload.CTMethod, res.CallType)
+	require.Equal(t, server.GlobalCaller(), res.Caller)
+	assert.Equal(t, flagsCaller.Interference, res.CallFlags.GetInterference())
+	assert.Equal(t, flagsCaller.State, res.CallFlags.GetState())
```

Comment:
why not `assert.Equal(t, flagsCaller, res.CallFlags)` ?

Reply:
check also CallOutgoing, please

Reply:
because these are different structures(flagsCaller, res.CallFlags)